openwisp_radius/apps.py

```diff
@@ -62,6 +64,13 @@ def connect_signals(self):
         RadiusToken = load_model('RadiusToken')
         RadiusAccounting = load_model('RadiusAccounting')
         User = get_user_model()
+        from openwisp_radius.api.freeradius_views import AccountingView
+
+        radius_accounting_success.connect(
+            radius_accounting_success_handler,
+            sender=AccountingView,
+            dispatch_uid='radius_accounting_success',
```

Comment:
`radius_accounting_success` is too generic, we should use something more specific, eg: `send_email_on_new_accounting`